frontend/src/components/Chat.jsx

```diff
@@ -15,7 +36,7 @@ class Chat extends Component {
   submitMessage = messageString => {
     // on submitting the ChatInput form, send the message, add it to the list and reset the input
     const message = { name: this.state.name, message: messageString };
```

Comment:
can we change message.message to message.text? @JasperCheung

Reply:
yes, that's makes it more reable i think.